rotkehlchen/chain/ethereum/modules/ens/decoder.py

```diff
@@ -139,17 +140,25 @@ def _decode_name_renewed(self, context: DecoderContext) -> DecodingOutput:
             return DEFAULT_DECODING_OUTPUT
 
         name = decoded_data[0]
-        amount = from_wei(decoded_data[1])
+        name_cost = from_wei(decoded_data[1])
```

Comment:
`name_cost` is confusing. As this is not the actual cost. It can be less if you reduce the refund.

```suggestion
        renewal_full_amount = from_wei(decoded_data[1])
```

Reply:
This is the actual cost of the name. `name_cost` = `full amount spent` - `refund amount`